rocks-lib/src/package/version.rs

```diff
@@ -243,29 +244,26 @@ impl PackageVersionReq {
             (PackageVersionReq::SemVer(version_req), PackageVersion::SemVer(semver)) => {
                 version_req.matches(&semver.version)
             }
-            (PackageVersionReq::SemVer(version_req), PackageVersion::DevVer(..)) => {
-                let large_version = Version {
-                    major: u64::MAX,
-                    minor: u64::MAX,
-                    patch: u64::MAX,
-                    pre: Prerelease::EMPTY,
-                    build: BuildMetadata::EMPTY,
-                };
-                version_req.matches(&large_version)
-            }
+            (PackageVersionReq::SemVer(..), PackageVersion::DevVer(..)) => false,
             (PackageVersionReq::Dev(..), PackageVersion::SemVer(..)) => false,
             (PackageVersionReq::Dev(name_req), PackageVersion::DevVer(devver)) => {
                 name_req.ends_with(&devver.modrev)
             }
+            (PackageVersionReq::Any, _) => true,
```

Comment:
question: should `(_, Any)` also be handled here?

Reply:
there's no `PackageVersion::Any`.